phantazm-zombies/src/main/java/com/github/phantazmnetwork/zombies/equipment/Equipment.java

```diff
@@ -1,11 +1,26 @@
 package com.github.phantazmnetwork.zombies.equipment;
 
+import net.minestom.server.entity.Player;
+
+/**
+ * Represents a piece of equipment that {@link Player}s may use.
+ */
 public interface Equipment {
 
+    /**
+     * Invoked when a {@link Player} changes their slot to or from the one containing this {@link Equipment}.
+     * @param selected Whether the {@link Player} is now holding this {@link Equipment}
```

Comment:
unnecessary IMO to bother linking the interface you're currently in

Reply:
(nitpick lvl 9000, doesn't matter if you change or not)

Reply:
oh well I did it everywhere